_data/sitemap.yml

```diff
@@ -26,8 +26,8 @@
     - title: Pricing
       url: /pricing/
 
-    - title: Docs
-      url: "https://docs.gruntwork.io"
+    - title: Deployment Guides
+      url: /guides
```

Comment:
Does "Production Deployment Guides" fit?

Is this `sitemap.yml` used solely for the footer? Or anywhere else?

Reply:
sitemap.yml is used solely in the footer. Not used elsewhere.

Since we have been using deployment guides :). Maybe just `Guides`?

Reply:
> sitemap.yml isn't used elsewhere

Roger

> Since we have been using deployment guides :). Maybe just Guides?

The typical website visitor would have no idea what "Guides" means and probably would never click. "Production Deployment Guides" at least gives some hint. Probably better SEO-wise too.

Reply:
Hm, when I asked if "Production Deployment Guides" fits, I meant "fits on one line like all other footer items." I can see now it clearly does not fit:

![Screen Shot 2019-09-19 at 10 01 48 PM](https://user-images.githubusercontent.com/711908/65280893-345a8380-db29-11e9-9e9f-cb04fa21babe.png)

I'll submit a PR to change to Deployment Guides...